package.json

```diff
@@ -16,6 +16,7 @@
     "dotenv": "^16.0.1",
     "get-function-params": "^2.0.7",
     "glob": "^8.0.3",
+    "oas-normalize": "^8.4.0",
```

Comment:
would it be safer to use the exact version number? or is relying on semver safe for this package?

Reply:
Tbh i'm happy to rely on ^ versions in all packages, the fixed versions are what the package-lock.json is for.